pkg/odo/cli/deploy/deploy.go

```diff
@@ -151,7 +151,7 @@ func NewCmdDeploy(name, fullName string) *cobra.Command {
 	clientset.Add(deployCmd, clientset.INIT, clientset.DEPLOY)
 
 	// Add a defined annotation in order to appear in the help menu
-	deployCmd.Annotations["command"] = "utility"
+	deployCmd.Annotations["command"] = "main"
```

Comment:
Aren't you facing the nil map panic here? Also, why not add this annotation when declaring `deployCmd` itself on L141?

Reply:
Nope, not facing it here since the annotations are propagated already with Validate / Run.